dpnp/tests/test_product.py

```diff
@@ -598,6 +599,8 @@ class TestMatmul:
     def setup_method(self):
         numpy.random.seed(42)
 
+    # Use both integer and float, to test both MKL and dpctl backends
+    @pytest.mark.parametrize("dtype", [numpy.int32, numpy.float32])
```

Comment:
Probably it would be good to create a list and declare it at the file beginning to pass ot at every place where required instead of repeating the same `[numpy.int32, numpy.float32]` and comment.
While comment can be placed at the list declaration to state the intention.